databases/week2/more-queries.sql

```diff
@@ -0,0 +1,24 @@
+USE hyf;
+select * from task;
+
+-- 1 
+SELECT  user.name, user.email FROM user
+WHERE user.email REGEXP '@spotify.com$';
+
+-- 2
+SELECT * FROM task
+JOIN user 
+ON task.id = user.id
```

Comment:
The id column of the table task has no relation at all with the id column of the user table. You need a foreign key to connect different tables. For instance, in the task table there is the status_id column that is a foreign key connected to the id column of the table status, such that a status_id of 2 in the table task refers to a status with id=2 in the table status. Does this make sense?